### PR TITLE
Merge the androidx to master

### PR DESCRIPTION
**Pull request by thagikura**
Now Android Studio 3.2 and the androidx 1.0.0 versions are stable.
Merging the androidx features to the master is ready.

### CODE REVIEW

**Comment by thagikura** [comment]
@MGaetan89 please confirm I'm going to merge the commits including yours into master. (Just in case because it's the same repository).

@yaraki Could you proof read?

File: flexbox/src/androidTest/java/com/google/android/flexbox/test/TestViewHolder.kt [hunk 1]
```diff
@@ -16,13 +16,12 @@
...
 
-import android.support.v7.widget.RecyclerView
 import android.view.View
 import android.widget.TextView
 
 /**
  * ViewHolder implementation for a flex item for testing.
  */
-internal class TestViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+internal class TestViewHolder(itemView: View) : androidx.recyclerview.widget.RecyclerView.ViewHolder(itemView) {
```
Comment (MGaetan89) [inline flexbox/src/androidTest/java/com/google/android/flexbox/test/TestViewHolder.kt]: Add an import for `androidx.recyclerview.widget.RecyclerView`

**Comment by MGaetan89** [comment]
I just had a minor comment, but it should be good otherwise.
I was using the beta version on a test app for a while without any issues.

File: flexbox/src/androidTest/java/com/google/android/flexbox/test/TestViewHolder.kt (hunk 1 above)
Comment (thagikura) [reply flexbox/src/androidTest/java/com/google/android/flexbox/test/TestViewHolder.kt]: Done. Thanks.

**Comment by thagikura** [comment]
Thanks. Merged.

**Pull request merged**